tests/heap/test_find_fake_fast.py

```diff
@@ -0,0 +1,144 @@
+import re
+
+import gdb
+
+import pwndbg
+import tests
+
+HEAP_FIND_FAKE_FAST = tests.binaries.get("heap_find_fake_fast.out")
+
+target_address = None
+
+
+def check_result(result, expected_size):
+    matches = re.findall(r"\bAddr: (0x[0-9a-f]+)", result)
+    assert len(matches) == 1
+    addr = int(matches[0], 16)
+
+    matches = re.findall(r"\bsize: (0x[0-9a-f]+)", result)
+    assert len(matches) == 1
+    size = int(matches[0], 16)
+
+    assert size == expected_size
+
+    # The chunk can't start too close to the target address
+    assert addr <= target_address - 0x10
+
+    # Clear the flags
+    size &= ~0xF
+
+    # The chunk should overlap the target address
+    assert addr + size > target_address
+
+
+def check_no_results(result):
+    matches = re.findall(r"\bAddr: (0x[0-9a-f]+)", result)
+    assert len(matches) == 0
+
+
+def test_find_fake_fast_command(start_binary):
+    global target_address
+
+    start_binary(HEAP_FIND_FAKE_FAST)
+    gdb.execute("break break_here")
+    gdb.execute("continue")
+
+    # Ensure memory at fake_chunk's heap_info struct isn't mapped.
+    unmapped_heap_info = pwndbg.heap.ptmalloc.heap_for_ptr(pwndbg.symbol.address("fake_chunk"))
+    assert pwndbg.gdblib.memory.peek(unmapped_heap_info) is None
+
+    # A gdb.MemoryError raised here indicates a regression from PR #1145
+    gdb.execute("find_fake_fast (void*)&fake_chunk+0x70")
+
+    result = gdb.execute("p/x &target_address", to_string=True)
+    matches = re.findall(r"\$1 = (0x[0-9a-f]+)", result)
+    assert len(matches) == 1
+    target_address = int(matches[0], 16)
```

Comment:
Any reason not to use `pwndbg.symbol.address()`?

Reply:
I got so used to running `gdb.execute` that I forgot I should just do that :P Will fix it.